MANIFEST.in

```diff
@@ -1,5 +1,6 @@
 graft src
 include *.md pipx_demo.gif logo.png LICENSE
+include *.toml
```

Comment:
It’s better spell out the file name explicitly since this may cause other TOML files to be bundled accidentally. But do we need to include the analyser configuration in sdist in the first place?

Reply:
Yes, I will make it .deepsource.toml`. But I have added this because the Test/lint were failing.

Reply:
If this is only to prevent test/lint from failing, you can always specifically `exclude` the file.